transaction-status/src/lib.rs

```diff
@@ -1074,7 +1074,7 @@ impl ConfirmedTransactionWithStatusMeta {
     }
 }
 
-#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
I added this in the same `PR` as it was a small addition which seems like it was just missed by `anza` and it prevented us from returning it from our RPC methods (which require structs used as return values implement `Clone`).